sample-data/instances/semantic-web-primer.json

```diff
@@ -1,53 +1,73 @@
 {
-  "id" : "5bf370e0-8cca-4d9c-82e4-5170ab2a0a39",
-  "source" : "manual",
```

Comment:
What was a `manual` source?

Reply:
I don't know. I think it was a (potentially unofficial) precursor to the `FOLIO` source. I think it referred to the record being manually created in the inventory UI and has no source record.